### PR TITLE
update Group IB TIA playbooks for solutions package + logo + sample data

### PR DESCRIPTION
**Pull request by ivanovchinnikov**
Fixes #

## Proposed Changes

  - Logo file added
  - Sample data added
  - Playbooks updated to the latest version

### CODE REVIEW

**Comment by v-jayakal** [comment]
@sarah-yo - Can you please review the PR? Thanks.

**Comment by sarah-yo** [comment]
Assigning @preetikr as it's solutions

File: Solutions/Group-IB/Playbooks/GIBTIA_APT_ThreatActor.json [hunk 1]
```diff
@@ -559,14 +559,14 @@
         "$connections": {
             "value": {
                 "azureloganalyticsdatacollector": {
-                    "connectionId": "/subscriptions/xx-xxx-xxxx-xxxx-xxx/resourceGroups/Group-IB-WS/providers/Microsoft.Web/connections/",
-                    "connectionName": "",
-                    "id": "/subscriptions/xx-xxx-xxxx-xxxx-xxx/providers/Microsoft.Web/locations/westeurope/managedApis/azureloganalyticsdatacollector"
+                    "connectionId": "/subscriptions/6a021fde-5198-441e-9190-9d634e1f4a84/resourceGroups/Group-IB-WS/providers/Microsoft.Web/connections/azureloganalyticsdatacollector-2",
```
Comment (sarah-yo) [inline Solutions/Group-IB/Playbooks/GIBTIA_APT_ThreatActor.json:562]: hardcoded subscription ID

File: Solutions/Group-IB/Playbooks/GIBTIA_APT_ThreatActor.json [hunk 2]
```diff
@@ -559,14 +559,14 @@
...
+                    "connectionName": "azureloganalyticsdatacollector-2",
+                    "id": "/subscriptions/6a021fde-5198-441e-9190-9d634e1f4a84/providers/Microsoft.Web/locations/westeurope/managedApis/azureloganalyticsdatacollector"
                 },
                 "azuremonitorlogs": {
-                    "connectionId": "/subscriptions/xx-xxx-xxxx-xxxx-xxx/resourceGroups/Group-IB-WS/providers/Microsoft.Web/connections/",
-                    "connectionName": "",
-                    "id": "/subscriptions/xx-xxx-xxxx-xxxx-xxx/providers/Microsoft.Web/locations/westeurope/managedApis/azuremonitorlogs"
+                    "connectionId": "/subscriptions/6a021fde-5198-441e-9190-9d634e1f4a84/resourceGroups/Group-IB-WS/providers/Microsoft.Web/connections/azuresentinel-Close-SentinelIncident-from-SNOW",
```
Comment (sarah-yo) [inline Solutions/Group-IB/Playbooks/GIBTIA_APT_ThreatActor.json:567]: hardcoded subscription ID

File: Solutions/Group-IB/Playbooks/GIBTIA_Attacks_ddos.json [hunk 3]
```diff
@@ -124,7 +124,7 @@
                         "resourcegroups": "Group-IB-WS",
                         "resourcename": "Group-IB",
                         "resourcetype": "Log Analytics Workspace",
-                        "subscriptions": "xx-xxx-xxxx-xxxx-xxx",
+                        "subscriptions": "6a021fde-5198-441e-9190-9d634e1f4a84",
```
Comment (sarah-yo) [inline Solutions/Group-IB/Playbooks/GIBTIA_Attacks_ddos.json:127]: hardcoded subscription ID

File: Solutions/Group-IB/Playbooks/GIBTIA_Attacks_ddos.json [hunk 4]
```diff
@@ -439,7 +439,7 @@
                                     "host": {
                                         "triggerName": "Batch_messages",
                                         "workflow": {
-                                            "id": "/subscriptions/xx-xxx-xxxx-xxxx-xxx/resourceGroups/Group-IB-WS/providers/Microsoft.Logic/workflows/GIBIndicatorProcessor"
+                                            "id": "/subscriptions/6a021fde-5198-441e-9190-9d634e1f4a84/resourceGroups/Group-IB-WS/providers/Microsoft.Logic/workflows/GIBIndicatorProcessor"
```
Comment (sarah-yo) [inline Solutions/Group-IB/Playbooks/GIBTIA_Attacks_ddos.json:442]: hardcoded subscription ID

File: Solutions/Group-IB/Playbooks/GIBTIA_Attacks_ddos.json [hunk 5]
```diff
@@ -636,14 +640,14 @@
         "$connections": {
             "value": {
                 "azureloganalyticsdatacollector": {
-                    "connectionId": "/subscriptions/xx-xxx-xxxx-xxxx-xxx/resourceGroups/Group-IB-WS/providers/Microsoft.Web/connections/",
-                    "connectionName": "",
-                    "id": "/subscriptions/xx-xxx-xxxx-xxxx-xxx/providers/Microsoft.Web/locations/westeurope/managedApis/azureloganalyticsdatacollector"
+                    "connectionId": "/subscriptions/6a021fde-5198-441e-9190-9d634e1f4a84/resourceGroups/Group-IB-WS/providers/Microsoft.Web/connections/azureloganalyticsdatacollector-2",
```
Comment (sarah-yo) [inline Solutions/Group-IB/Playbooks/GIBTIA_Attacks_ddos.json:643]: hardcoded subscription ID

File: Solutions/Group-IB/Playbooks/GIBTIA_Attacks_ddos.json [hunk 6]
```diff
@@ -636,14 +640,14 @@
...
+                    "connectionName": "azureloganalyticsdatacollector-2",
+                    "id": "/subscriptions/6a021fde-5198-441e-9190-9d634e1f4a84/providers/Microsoft.Web/locations/westeurope/managedApis/azureloganalyticsdatacollector"
                 },
                 "azuremonitorlogs": {
-                    "connectionId": "/subscriptions/xx-xxx-xxxx-xxxx-xxx/resourceGroups/Group-IB-WS/providers/Microsoft.Web/connections/",
-                    "connectionName": "",
-                    "id": "/subscriptions/xx-xxx-xxxx-xxxx-xxx/providers/Microsoft.Web/locations/westeurope/managedApis/azuremonitorlogs"
+                    "connectionId": "/subscriptions/6a021fde-5198-441e-9190-9d634e1f4a84/resourceGroups/Group-IB-WS/providers/Microsoft.Web/connections/azuresentinel-Close-SentinelIncident-from-SNOW",
```
Comment (sarah-yo) [inline Solutions/Group-IB/Playbooks/GIBTIA_Attacks_ddos.json:648]: hardcoded subscription ID

File: Solutions/Group-IB/Playbooks/GIBTIA_Attacks_deface.json [hunk 7]
```diff
@@ -124,7 +124,7 @@
                         "resourcegroups": "Group-IB-WS",
                         "resourcename": "Group-IB",
                         "resourcetype": "Log Analytics Workspace",
-                        "subscriptions": "xx-xxx-xxxx-xxxx-xxx",
+                        "subscriptions": "6a021fde-5198-441e-9190-9d634e1f4a84",
```
Comment (sarah-yo) [inline Solutions/Group-IB/Playbooks/GIBTIA_Attacks_deface.json:127]: hardcoded subscription ID

File: Solutions/Group-IB/Playbooks/GIBTIA_Attacks_deface.json [hunk 8]
```diff
@@ -403,7 +403,7 @@
                                     "host": {
                                         "triggerName": "Batch_messages",
                                         "workflow": {
-                                            "id": "/subscriptions/xx-xxx-xxxx-xxxx-xxx/resourceGroups/Group-IB-WS/providers/Microsoft.Logic/workflows/GIBIndicatorProcessor"
+                                            "id": "/subscriptions/6a021fde-5198-441e-9190-9d634e1f4a84/resourceGroups/Group-IB-WS/providers/Microsoft.Logic/workflows/GIBIndicatorProcessor"
```
Comment (sarah-yo) [inline Solutions/Group-IB/Playbooks/GIBTIA_Attacks_deface.json:406]: hardcoded subscription ID

File: Solutions/Group-IB/Playbooks/GIBTIA_Attacks_deface.json [hunk 9]
```diff
@@ -575,8 +575,12 @@
                 "defaultValue": "https://bt.group-ib.com/api/v2/",
                 "type": "String"
             },
+            "GIB Attacks Deface URL Action": {
+                "defaultValue": "unknown",
+                "type": "String"
+            },
             "GIB Username": {
-                "defaultValue": "Xxxx@domain.com",
+                "defaultValue": "i.ovchinnikov+2@group-ib.com",
```
Comment (sarah-yo) [inline Solutions/Group-IB/Playbooks/GIBTIA_Attacks_deface.json:583]: hardcoded username

File: Solutions/Group-IB/Playbooks/GIBTIA_Attacks_deface.json [hunk 10]
```diff
@@ -598,14 +602,14 @@
         "$connections": {
             "value": {
                 "azureloganalyticsdatacollector": {
-                    "connectionId": "/subscriptions/xx-xxx-xxxx-xxxx-xxx/resourceGroups/Group-IB-WS/providers/Microsoft.Web/connections/",
-                    "connectionName": "",
-                    "id": "/subscriptions/xx-xxx-xxxx-xxxx-xxx/providers/Microsoft.Web/locations/westeurope/managedApis/azureloganalyticsdatacollector"
+                    "connectionId": "/subscriptions/6a021fde-5198-441e-9190-9d634e1f4a84/resourceGroups/Group-IB-WS/providers/Microsoft.Web/connections/azureloganalyticsdatacollector-2",
```
Comment (sarah-yo) [inline Solutions/Group-IB/Playbooks/GIBTIA_Attacks_deface.json:605]: hardcoded subscription ID

File: Solutions/Group-IB/Playbooks/GIBTIA_Attacks_deface.json [hunk 11]
```diff
@@ -598,14 +602,14 @@
...
+                    "connectionName": "azureloganalyticsdatacollector-2",
+                    "id": "/subscriptions/6a021fde-5198-441e-9190-9d634e1f4a84/providers/Microsoft.Web/locations/westeurope/managedApis/azureloganalyticsdatacollector"
                 },
                 "azuremonitorlogs": {
-                    "connectionId": "/subscriptions/xx-xxx-xxxx-xxxx-xxx/resourceGroups/Group-IB-WS/providers/Microsoft.Web/connections/",
-                    "connectionName": "",
-                    "id": "/subscriptions/xx-xxx-xxxx-xxxx-xxx/providers/Microsoft.Web/locations/westeurope/managedApis/azuremonitorlogs"
+                    "connectionId": "/subscriptions/6a021fde-5198-441e-9190-9d634e1f4a84/resourceGroups/Group-IB-WS/providers/Microsoft.Web/connections/azuresentinel-Close-SentinelIncident-from-SNOW",
```
Comment (sarah-yo) [inline Solutions/Group-IB/Playbooks/GIBTIA_Attacks_deface.json:610]: hardcoded subscription ID

File: Solutions/Group-IB/Playbooks/GIBTIA_Attacks_phishing.json [hunk 12]
```diff
@@ -124,7 +124,7 @@
                         "resourcegroups": "Group-IB-WS",
                         "resourcename": "Group-IB",
                         "resourcetype": "Log Analytics Workspace",
-                        "subscriptions": "xx-xxx-xxxx-xxxx-xxx",
+                        "subscriptions": "6a021fde-5198-441e-9190-9d634e1f4a84",
```
Comment (sarah-yo) [inline Solutions/Group-IB/Playbooks/GIBTIA_Attacks_phishing.json:127]: hardcoded subscription ID

**Review by sarah-yo — CHANGES_REQUESTED** [review]
Many hardcoded subscription ID and usernames, there were too many to mark up. Please remove.

Please remove the "en-us" part of the URL in your readme

Please put the playbook title back in for each deployment button in the readme

**Review by sarah-yo — CHANGES_REQUESTED** [review]
Please put the deploy buttons next to the relevant playbook in the Group-IB Sentinel Playbooks Collections Detailed Description part of the readme

**Pull request merged**